docs/docker-images/buildkit-standalone.md

```diff
@@ -0,0 +1,61 @@
+
+If you are using this container locally, please note that `EARTHLY_BUILDKIT_HOST` values where the host is `127.0.0.1`, ` ::1/128`, or `localhost` are considered local and will result in Earthly attempting to manage the buildkit container itself. Consider using your hostname, or another alternative name in these cases.
+
+### Supported Environment Variables
+
+| Variable Name                       | Default Values                 | Description                                                                                                                                                                   |
+|-------------------------------------|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| EARTHLY_ADDITIONAL_BUILDKIT_CONFIG  |                                | Additional `buildkitd` config to append to the generated configuration file.                                                                                                  |
+| BUILDKIT_LOCAL_REGISTRY_LISTEN_PORT | `8371`                         | What port should the internal cache registry listen on?                                                                                                                       |
+| BUILDKIT_TCP_TRANSPORT_ENABLED      |                                | Set to `true` when the `buildkitd` instance is going to be used remotely                                                                                                      |
+| BUILDKIT_TLS_ENABLED                |                                | Set to `true` when the `buildkittd` instance will require mTLS from the clients. You will also need to mount certificates into the right place (`/etc/*.pem`).                |
+| CNI_MTU                             | MTU of first default interface | Set this when we autodetect the MTU incorrectly. The device used for autodetection can be shown by the command  `ip route show \| grep default \| cut -d' ' -f5 \| head -n 1` |
+| EARTHLY_RESET_TMP_DIR               | `false`                        | Cleans out `EARTHLY_TMP_DIR` before running, if set to `true`. Useful when you host-mount an temp dir across runs.                                                            |
+| NETWORK_MODE                        | `cni`                          | Specifies the networking mode of `buildkitd`. Default uses a CNI bridge network, configured with the `CNI_MTU`.                                                               |
+| EARTHLY_TMP_DIR                     | `/tmp/earthly`                 | Specifies the location of `earthly`s temp dir. You can also mount an external volume to this path to preserve the contents across runs.                                       |
```

Comment:
This reminds me - we should recommend people to mount a Docker volume to this dir almost all the time when using buildkitd (buildkit image and earthly image with internal buildkit).

Reason: If the underlying FS does not support overlayfs on top of it then buildkit will switch to the dumb snapshoter (I think it's called native?), which instead of using copy-on-write layers, it just copies the entire file system on every single layer. This makes disk space run out really fast, which is very inconvenient.

If you don't mount anything there, then it's basically overlayfs on top of overlayfs, which is not supported.

For the most part, only `ext4` and Docker volumes are supported. So most of the time, on Linux you could mount a host volume, but on Mac and Windows, the host volume would not work with overlayfs. But Docker volumes are supported in any situation.

Reply:
Added a section with less detail.

Reply:
It may be best to

1. Change the quick examples to always include a Docker volume mount - not a host mount.
2. Include enough details to make the user understand that they should mount something here almost always.

If people get issues related to disk space or aggressive GCing, it can be very hard for us to figure out that they are using a bad underlying FS. It can lead to a very bad user experience with little signal that this is the issue.